roles/operator_bundle_scorecard_tests/files/scorecard-basic-config.yml

```diff
@@ -7,14 +7,14 @@ metadata:
 stages:
   - parallel: true
     tests:
-      - image: quay.io/operator-framework/scorecard-test:latest
+      - image: quay.io/operator-framework/scorecard-test@sha256:981e2acb40b53954a7daadbe1114af1faaa51a632cf7654765252124e50d7c7d
         entrypoint:
           - scorecard-test
           - basic-check-spec
         labels:
           suite: basic
           test: basic-check-spec-test
-      - image: quay.io/operator-framework/scorecard-test:latest
+      - image: quay.io/operator-framework/scorecard-test@sha256:981e2acb40b53954a7daadbe1114af1faaa51a632cf7654765252124e50d7c7d
```

Comment:
Is it possible to come up with a tagging scheme for this image instead of referring to a digest?

Reply:
Bill Peck had asked that I make this point the the sha256 value instead so that the config works with clusters in disconnected environments

Reply:
It is very unlikely this playbook will be used in disconnected env.
please revert back to  cause we have to follow recommendation on the top of this file and in the readme
https://github.com/redhat-openshift-ecosystem/operator-test-playbooks/blob/master/roles/operator_bundle_scorecard_tests/README.md

And I cannot se digest in https://github.com/redhat-openshift-ecosystem/operator-scorecard-test-container/blob/main/scorecard-basic-config.yml

Reply:
Fixed.  It also looks like the scorecard-test:v1.9.0 image was updated and the sha was changed